src/sql-parser/tests/testdata/ddl

```diff
@@ -891,16 +891,16 @@ CREATE INDEX IF EXISTS myschema.ind ON foo(b)
 parse-statement
 DROP DATABASE mydb
 ----
-DROP DATABASE mydb
+DROP DATABASE mydb CASCADE
```

Comment:
idk if this behavior is whack or not -- but let me know and I can change it.  Could easily change it to only print non-default (`RESTRICT`) or always print.

This doesn't seem to be a standard SQL behavior (`CASCADE` or `RESTRICT` on `DROP DATABASE`) so not sure what to default to.  Not sure if you have a preference

Reply:
Mentioned this above, but let's just follow whatever e.g. `DROP INDEX` does for printing its default/non-default options.